### PR TITLE
chore: add and configure environment_indicator

### PR DESCRIPTION
**Pull request by lazysoundsystem**
Refs: OPS-8939

### CODE REVIEW

File: config/environment_indicator.switcher.local.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+langcode: en
+status: true
+dependencies: {  }
+machine: local
+description: null
+name: Local
+weight: ''
+url: 'https://numbers.test'
+fg_color: '#000000'
+bg_color: '#ffffff'
```
Comment (rupl) [inline config/environment_indicator.switcher.local.yml:11]:
```suggestion
fg_color: '#ffffff'
bg_color: '#000000'
```

Has this black-on-white been deployed many places for locals? If possible I'd prefer to keep it as core has: with white-on-black.

**Review by rupl — APPROVED** [review]
Approved with the inline comments/suggestions. But those comments depend on what is happening across other projects.

I have a preference to hold onto the white-on-black that core provides for the local environments. After so many years of having it that way, the swapped colors bother me.

**Comment by lazysoundsystem** [comment]
Yup, no problem to make that switch, I'll include it in the other PRs too.

Note that these settings are without permissions so it should only be user 1 that sees the switcher - permissions will need to be configured on a site-by-site basis depending on use and roles. 

And the indicator colors will show on local, but will need ansible settings to be displayed on dev and prod.

**Comment by rupl** [comment]
Good to know, and thanks for honoring my pickiness 😁

**Pull request merged**